freifunk-berlin/falter-packages
# Some gardening

Kompiliert: ja https://buildbot.berlin.freifunk.net/#/builders/2/builds/183
Läuft live: nicht getestet

Beschreibung deiner Änderungen:

- Merge firewall-defaults package into falter-more (see #384 for context)
- Bump version of falter-common and the variants
- Enable 23.05 branch in build.sh
- Remove unused scripts
- Update build targets lists
- Update tunspace TODOs
